src/views/Dashboard/DashboardV2.js

```diff
@@ -475,7 +480,20 @@ export default function DashboardV2() {
                 {feeHistory.length ? (
                   <div className="App-card-row">
                     <div className="label">Fees since {formatDate(feeHistory[0].to)}</div>
-                    <div>${formatAmount(currentFeesUsd, USD_DECIMALS, 2, true)}</div>
+                    <div>
+                      <TooltipComponent
+                        position="right-bottom"
+                        className="nowrap"
+                        handle={`$${formatAmount(totalCurrentFees, USD_DECIMALS, 2, true)}`}
+                        renderContent={() => (
+                          <>
+                            Distributed Fees: ${formatAmount(currentFeesUsd, USD_DECIMALS, 2, true)}
+                            <br />
+                            Spread Capture: ${formatAmount(currentWeeksMMFees, USD_DECIMALS, 2, true)}
+                          </>
+                        )}
+                      />
+                    </div>
```

Comment:
The numbers here don't quite add up in the UI, is there a small disparity on purpose? 

![image](https://user-images.githubusercontent.com/19278287/186827351-47f8b00d-3c76-4125-87cd-8de8f35f4410.png)

Reply:
lol https://github.com/mycelium-ethereum/swaps-client/pull/115/files#diff-bf29c66c1d51657e91e65f4fd1b1cb1ab36f14b5a3df3dae557b9c0cd47a3f25R152